platform-core-module/src/main/java/com/softicar/platform/core/module/user/password/UserPasswordGenerator.java

```diff
@@ -54,10 +56,17 @@ public void resetUserPassword(AGUser user) {
 		try (DbTransaction transaction = new DbTransaction()) {
 			String password = new UserPasswordGenerator().generatePassword();
 			new UserPasswordUpdater(user, password).update();
-			AGUser.sendPaswordResetNotification(user, password);
+			if (AGCoreModuleInstance.getInstance().getEmailServer() != null) {
+				AGUser.sendPaswordResetNotification(user, password);
+			}
```

Comment:
shouldn't we assume that the email server is configures? otherwise, the whole password reset mechanism would not work